scripts/ci/jobs/openshift_4_operator_e2e_tests.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env -S python3 -u
+
+"""
+Run operator e2e tests in a openshift 4 cluster provided via a hive cluster_claim.
```

Comment:
Do you documentation on this? Honestly I don't understand it.

Reply:
I'm just trying to be consistent with the another script in the same directory from which I cargo-culted.